opentelemetry/proto/common/v1/common.proto

```diff
@@ -32,8 +62,8 @@ message AnyValue {
     bool bool_value = 2;
     int64 int_value = 3;
     double double_value = 4;
-    ArrayValue array_value = 5;
-    KeyValueList kvlist_value = 6;
+    ArrayValue array_value = 5 [(field_opts).is_json = true];
```

Comment:
What does this annotation do?

Reply:
This annotation tells the parquet schema generator to stop recursing and treat the field as a JSON data blob, to be serialized as one column. Otherwise, we are stuck in a recursive loop with parquet, which requires a flat schema.

Reply:
Can you also add this explanation to the code? More details will be event better, since I am not sure why it is supposed to recurse and why it needs to stop.